polyaxon/experiments/views.py

```diff
@@ -202,3 +208,20 @@ class ExperimentJobStatusDetailView(ExperimentJobViewMixin, RetrieveUpdateAPIVie
     serializer_class = ExperimentJobStatusSerializer
     permission_classes = (IsAuthenticated,)
     lookup_field = 'uuid'
+
+
+class ExperimentLogsView(ExperimentViewMixin, APIView):
```

Comment:
This could use the `RetrieveAPIView`

```python
class ExperimentLogsView(ExperimentViewMixin, RetrieveAPIView):
    permission_classes = (IsAuthenticated,)

    def get(self, request, *args, **kwargs):
          ...
```